frontend/src/metabase/pulse/components/PulseCardPreview.jsx

```diff
@@ -114,7 +114,7 @@ export default class PulseCardPreview extends Component {
         </div>
         <div
           className="bordered rounded bg-white scroll-x"
-          style={{ display: !cardPreview && "none" }}
+          style={{ display: !cardPreview && "none", padding: "16px" }}
```

Comment:
I removed the padding around static viz images to work better with the new email template, so I'm adding it back here for pulse previews on the FE.